src/main.rs

```diff
@@ -9,9 +9,15 @@
 // specific language governing permissions and limitations under the License.
 
 extern crate clap;
+#[macro_use] extern crate nickel;
```

Comment:
This lets us use the `middleware` macro which expands out to something like this during compilation: https://github.com/nickel-org/nickel.rs/issues/346#issuecomment-225222662.